### DISCUSSION TITLE
/hyndsight/badarima/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#307] [Announcements]
Why doesn't auto.arima() return the model with the lowest AICc value? 
 https://robjhyndman.com/hyndsight/badarima/

**Comment by robjhyndman** [comment]
```header
Originally posted by ss g on 2018-12-28 11:05:12
````
Thank you for your detailed explanation and now I have a deeper
understanding of auto.arima().\
But it occurred to me another question about model selections by AICc.\
Since AICc is bias-corrected version of the AIC, and minimising the AIC
is equivalent to minimising the **CV** value, but I think here CV means
for iid sample rather than **CV for time series**, is that correct or
make any sense? Is it safe to use AICc to approximate CV for time
series?

Thanks.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2018-12-30 00:48:15
````
In a time series context, AICc approximates the CV based on one-step
forecast errors.

**Reply by robjhyndman** [reply]
```header
Originally posted by ss g on 2018-12-30 14:58:28
````
Thank you very much. It\'s very helpful.

**Comment by khanhninhnguyen** [comment]
Sorry but did you check parameter redundancy in the auto.arima?
I used auto.arima, and I realized that the BIC of just a few models were printed when I put trace = TRUE.

**Comment by TekleAman** [comment]
Dear Professor,
I would like to forecast the energy consumption in hourly, daily and monthly use. Because the time series data contains mutiple seasonality which values should I use for the "m" in auto_arima()?. Thank you.

**Reply by robjhyndman** [reply]
Hourly energy consumption data will have seasonal periods of 24, 24*7 and 24*365.
